src/controllers/dashboardController.ts

```diff
@@ -15,6 +15,7 @@ export const userInfo = async (
       select: {
         name: true,
         phoneNumber: true,
+        batch: true,
```

Comment:
pls fix that `undefined` in the dashboard this time

Reply:
That has to be done in frontend, not backend.

Reply:
just reminding you for the 100th time :p